docs/release-cycle.md

```diff
@@ -0,0 +1,81 @@
+### kubeadm's release cycle and quarterly agenda
+
+Kubernetes has a well defined release process that improves with every release.
+kubeadm is a core component of Kubernetes.
```

Comment:
I don't think this statement is true (although the reverse is).

Reply:
I can remove this line; it's not needed for the doc